packages/web-components/src/presence-badge/README.md

```diff
@@ -0,0 +1,117 @@
+---
+id: card
+title: fluent-card
+sidebar_label: card
+custom_edit_url: https://github.com/microsoft/fluentui/edit/master/packages/web-components/src/card/README.md
+---
+
+# fluent-presence-badge
+
+## Overview
+
+A `fluent-presence-badge` is designed to be used with the Avatar control to display a status icon based on the in/out/busy status of an Office user.
+
+## API
+
+Extends FAST Element
+
+_Component Name_
+
+- `fluent-presence-badge`
+
+_Attributes_
+
+- `status` - string value that sets the icon to display: `"available" | "away" | "busy" | "dnd" | "outofoffice" | "offline" | "blocked"`
+- `outofoffice` - boolean value that shows user's in/out of office status and visually overrides the status icon for `"away"` and `"offline"` and switches other status icons to the fluent outline
+
+**Events**
+
+- None
+
+## Anatomy and appearance
+
+```html
+<!-- shadow root -->
+<div aria-hidden="true" class="presence-badge">
+  <slot name="busy">
+    <svg><path></path></svg>
+  </slot>
+</div>
+<!-- end shadow root -->
+```
+
+_Slot names_
+
+- available
+- away
+- busy
+- dnd
+- offline
+- unknown
+- blocked
+- outofoffice
+
+_Host attributes_
+
+status =
+
+- available
+- away
+- busy
+- dnd
+- offline
+- unknown
+- blocked
+- outofoffice
+
+## CSS overrides
+
+The badge can be customized several ways:
+
+- icon size and color can be changed
+  - `--presence-size: 16px` can be changed
+  - Color can be changed for each status:
+    - `--presence-color-available`
+    - `--presence-color-away`
+    - `--presence-color-busy`
+    - `--presence-color-do-no-disturb`
+    - `--presence-color-offline`
+    - `--presence-color-unknown`
+    - `--presence-color-blocked`
+    - `--presence-color-out-of-office`
+- icon mask width and color
+  - `--presence-mask-width: 8px;`
```

Comment:
Masking around badges is going away in some of the latest Fluent updates (on Windows) and since it's problematic with opacity, which we'll need to support soon, I suggest we remove it from the base component.